informal_update/writable_nested_serializers.py

```diff
@@ -0,0 +1,426 @@
+# -*- coding: utf-8 -*-
+# SOURCE: https://raw.githubusercontent.com/beda-software/drf-writable-nested/f32ad59/drf_writable_nested/mixins.py
+from collections import OrderedDict, defaultdict
+
+from django.contrib.contenttypes.fields import GenericRelation
+from django.contrib.contenttypes.models import ContentType
+from django.db.models import ProtectedError
+from django.core.exceptions import FieldDoesNotExist
+from django.db.models.fields.related import ForeignObjectRel
```

Comment:
This file should be in a common directory. Let's move this to main/writable_nested_serializers.py